src/app/pages/send/send-crypto-asset-form/family/bitcoin/components/bitcoin-send-max-button.tsx

```diff
@@ -0,0 +1,83 @@
+import { useCallback } from 'react';
+import toast from 'react-hot-toast';
+
+import { Box, Button, color } from '@stacks/ui';
+import { ButtonProps } from '@stacks/ui';
+import { SendCryptoAssetSelectors } from '@tests/selectors/send.selectors';
+import { useField } from 'formik';
+
+import { Money } from '@shared/models/money.model';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { Tooltip } from '@app/components/tooltip';
+
+const sendMaxTooltipLabel = 'This amount is affected by the fee you choose';
+
+interface BitcoinSendMaxButtonProps extends ButtonProps {
+  balance: Money;
+  isSendingMax?: boolean;
+  onSetIsSendingMax(value: boolean): void;
+  sendMaxBalance: string;
+  sendMaxFee?: string;
+}
+export function BitcoinSendMaxButton({
+  balance,
+  isSendingMax,
+  onSetIsSendingMax,
+  sendMaxBalance,
+  sendMaxFee,
+  ...props
+}: BitcoinSendMaxButtonProps) {
+  const [, _, amountFieldHelpers] = useField('amount');
+  const [, __, feeFieldHelpers] = useField('fee');
```

Comment:
Would be good to separate the styling of this button, from the data manipulation of the form.